src/server/serveractiveobject.h

```diff
@@ -220,6 +226,7 @@ class ServerActiveObject : public ActiveObject
 		  it could be confused to some other object by some client.
 		- This is usually set to true by the step() method when the object wants
 		  to be deleted but can be set by anything else too.
+		Note: Do not assign this directly, use markForRemoval() instead.
 	*/
 	bool m_pending_removal = false;
```

Comment:
Could you please make this either protected or private, if possible?

Reply:
This will require a `getThing() {return thing;}` which stinks but okay, I'll do it. C++ is a bad OOP language either way, there's no winning move here.